mne/utils/docs.py

```diff
@@ -4431,18 +4429,43 @@ def _reflow_param_docstring(docstring, has_first_line=True, width=75):
 smoothing increases with frequency.*
 Source: `FieldTrip tutorial: Time-frequency analysis using Hanning window,
 multitapers and wavelets <https://www.fieldtriptoolbox.org/tutorial/timefrequencyanalysis>`_.
+"""
+
+docdict["temporal-window_tfr_morlet_notes"] = (
+    _temporal_window_tfr_base
+    + r"""
+In MNE-Python, the length of the Morlet wavelet is affected by the arguments
+``freqs`` and ``n_cycles``, which define the frequencies of interest
+and the number of cycles, respectively. For the time-frequency representation,
+the length of the wavelet is defined such that both tails of
+the wavelet extend five standard deviations from the midpoint of its Gaussian
+envelope and that there is a sample at time zero.
+
+The length of the wavelet is thus :math:`10\times\mathtt{sfreq}\cdot\sigma-1`,
+which is equal to :math:`\frac{5}{\pi} \cdot \frac{\mathtt{n\_cycles} \cdot
+\mathtt{sfreq}}{\mathtt{freqs}} - 1`, where
+:math:`\sigma = \frac{\texttt{n\_cycles}}{2\pi f}` corresponds to the standard
```

Comment:
```suggestion
:math:`\sigma = \frac{\mathtt{n\_cycles}}{2\pi f}` corresponds to the standard
```
if you use `texttt` the backslash escape gets rendered